### PR TITLE
Sample Prometheus/Grafana setup

### PR DESCRIPTION
**Pull request by gustavclausen**
Ready for be expanded with some interesting metrics by @LasseRaatz and @emilbartholdy 🤩

**Changes:**
* Added simple 'latest' counter in simulator application.
* Added sample "coded" configuration for Grafana.
* Update README with Grafana URL and credentials.
* Add custom Grafana image push in pipeline (waiting for @DagBAndersen to create the registry on Docker Hub).

### CODE REVIEW

File: .github/workflows/ci-cd.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
           cd backend
```
Comment (gustavclausen) [inline .github/workflows/ci-cd.yml]: Apparently some formatting missing... What's interesting is the added `Push grafana to Dockerhub` step

File: docker-compose.prod.yml [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
-  
+
   frontend:
     image: dagandersen/frontend
+
+  grafana:
+    image: dagandersen/grafana
+    environment:
+      - GF_SECURITY_ADMIN_USER=Helge&Friends
+      - GF_SECURITY_ADMIN_PASSWORD=uiUIui # Super safe
```
Comment (emilbartholdy) [inline docker-compose.prod.yml]: 😄

File: .github/workflows/ci-cd.yml [hunk 3]
```diff
@@ -57,70 +57,78 @@ jobs:
...
+          workdir: frontend
 
-    - name: Push frontend to Dockerhub
-      uses: elgohr/Publish-Docker-Github-Action@master
-      with:
-        name: dagandersen/frontend
-        username: ${{ secrets.DOCKER_DAG_TEST_USER }}
-        password: ${{ secrets.DOCKER_DAG_TEST_PASSWORD }}
-        workdir: frontend
+      - name: Push grafana to Dockerhub
```
Comment (emilbartholdy) [inline .github/workflows/ci-cd.yml]: Why push grafana image to Dockerhub?

Comment (gustavclausen) [reply .github/workflows/ci-cd.yml]: Because we build a custom Grafana image where we bake in our configuration.

**Pull request merged**